source/_posts/2022-03-02-release-20223.markdown

```diff
@@ -816,6 +816,14 @@ The stale restored sensor can be safely removed.
 
 {% enddetails %}
 
+{% details "Workday" %}
+
+Work day now requires a 2-character [ISO 3166-1] country code instead of the country's name.
+
+[ISO 3166-1]: https://en.wikipedia.org/wiki/ISO_3166-1
+
```

Comment:
Got a reference/commit related to this?

Reply:
If it is a bug, we should fix it... not document it.

Reply:
I don't think it is truly a bug and we could be more permissive when checking for sure (our province key is actually not needed at all). Part of the problem is terminology around country and province.

Routes forward I see:
- Accept the current logic is reasonable, add the breaking change note
- Update integration to be more permissive. This should allow the integration to work just like before but is potentially less versatile. I did do this in: https://github.com/home-assistant/core/pull/67508, but changed my mind, should we re open?
- Do the above and then remove the redundant `province` key in the configuration and possibly change `country` to something more generic. This is a bigger breaking change that I don't think is the best option.

Let me know preferred route forward